pvlib/spa_c_files/README.md

```diff
@@ -1,34 +1,81 @@
 README
 ------
 
-NREL provides a C implementation of the solar position algorithm 
-described in 
-[Reda, I.; Andreas, A. (2003). Solar Position Algorithm for Solar Radiation Applications. 55 pp.; NREL Report No. TP-560-34302](
-http://www.nrel.gov/docs/fy08osti/34302.pdf).
+NREL provides a C implementation of the solar position algorithm described in
+[Reda, I.; Andreas, A. (2003). Solar Position Algorithm for Solar Radiation Applications. 55 pp.; NREL Report No. TP-560-34302](http://www.nrel.gov/docs/fy08osti/34302.pdf).
 
 This folder contains the files required to make NREL's C code accessible
 to the ``pvlib-python`` package. We use the Cython package to wrap NREL's SPA 
 implementation. 
 
-** Due to licensing issues, you must download the NREL C files from their 
-[website](http://www.nrel.gov/midc/spa) **
-
-Download the ``spa.c`` and ``spa.h`` files from NREL, 
-and copy them into the ``pvlib/spa_c_files`` directory. 
+** Due to licensing issues, the [SPA C files](http://www.nrel.gov/midc/spa) can
```

Comment:
http://www.nrel.gov/midc/spa forwards to https://midcdmz.nrel.gov/

looks like https://midcdmz.nrel.gov/spa/ is now the correct url